crates/fluvio-smartengine/Cargo.toml

```diff
@@ -12,11 +12,11 @@ description = "The official Fluvio SmartEngine"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-engine = ["wasmtime"]
+engine = ["wasmtime-engine"]
 wasi = ["wasmtime-wasi", "engine"]
 transformation = ["serde_json", "serde_yaml"]
 default = ["engine"]
-
+wasmtime-engine = ["wasmtime", "wasmtime-wasi"]
```

Comment:
with this update, `wasmtime-wasi` dependency will be pulled by default, could you update it to keep current behavior?

Reply:
Ok, this should be a mistake.